### PR TITLE
[Fix 2956] augeas table returns no data.

### PR DESCRIPTION
**Pull request by spasam**
[Fix 2490] Bundle augeas lenses in osquery distribution.
Pass osquery lenses path to aug_init() method.
Create and copy lenses to /usr/share/osquery/lenses/ on Linux.
Create and copy lenses to /private/var/osquery/lenses/ on OSX.
Also copy COPYING file from augeas to above directories.

Tested on Ubuntu 16.04 using deb package created after this change.

Unrelated change: Fixed hardware-monitoring.conf querypack.
Changed 'descriptor' to 'description'

### CODE REVIEW

File: osquery/tables/system/posix/augeas.cpp [hunk 1]
```diff
@@ -15,6 +15,12 @@
 #include <osquery/logger.h>
 #include <osquery/tables.h>
 
+#ifdef __APPLE__
+#define LENSES_PATH "/private//var/osquery/lenses"
```
Comment (ryanheffernan) [inline osquery/tables/system/posix/augeas.cpp]: is the extra '/' a typo?

**Comment by spasam** [comment]
@ryanheffernan Thanks for catching the copy-paste bug. Fixed double slash and updated review!

**Comment by theopolis** [comment]
This is good to go, but we'll need to install a copy of the GNU license along-side these lenses if we include them in our distribution. This is fine, we can move the `/usr/local/osquery/Cellar/augeas/*/COPYING` into the `LENSES_DST` directory.

**Comment by spasam** [comment]
:laughing: I was just looking at https://github.com/facebook/osquery/issues/2490 and fixing this. Will update in few mins.

**Comment by spasam** [comment]
@theopolis Done. Tested on Ubuntu 16.04. 
/usr/share/osquery/lenses/COPYING exists after installing deb package.

**Comment by theopolis** [comment]
ok to test

**Pull request merged**